From 4e2e7a78a2456635fc628848c7ca2a2d10679b9e Mon Sep 17 00:00:00 2001
From: maximilian attems <max@stro.at>
Date: Wed, 8 Sep 2010 16:21:52 +0800
Subject: [JOBS] Debug compile fix

No point in tracing a no longer undeclared "ps->cmd", fixes:
jobs.c: In function \u2018commandtext\u2019:
jobs.c:1192: error: \u2018ps\u2019 undeclared (first use in this function)
jobs.c:1192: error: (Each undeclared identifier is reported only once
jobs.c:1192: error: for each function it appears in.)

Signed-off-by: maximilian attems <max@stro.at>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
(cherry picked from commit 61a4c1de836712fdd40e288ac52b22de466bb886)

Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 ChangeLog  |    4 ++++
 src/jobs.c |    3 +--
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index f6a25a3..e02266f 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,7 @@
+2010-09-08  maximilian attems <max@stro.at>
+
+	* Debug compile fix.
+
 2010-09-08  Jilles Tjoelker <jilles@stack.nl>
 
 	* Fix varinit ordering that broke fc.
diff --git a/src/jobs.c b/src/jobs.c
index 060187c..826a9af 100644
--- a/src/jobs.c
+++ b/src/jobs.c
@@ -1189,8 +1189,7 @@ commandtext(union node *n)
 	STARTSTACKSTR(cmdnextc);
 	cmdtxt(n);
 	name = stackblock();
-	TRACE(("commandtext: name %p, end %p\n\t\"%s\"\n",
-		name, cmdnextc, ps->cmd));
+	TRACE(("commandtext: name %p, end %p\n", name, cmdnextc));
 	return savestr(name);
 }
 
-- 
1.7.4.1

